### PR TITLE
iOS example did not build on case-sensitive filesystem

### PR DESCRIPTION
**Pull request by texus**
The path to the example was hardcoded and contained "ios" while the folder name was "iOS".

### CODE REVIEW

**Comment by Ceylo** [comment]
👍

File: examples/iOS/CMakeLists.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 # All source files
 set(SRC ${SRCROOT}/main.cpp)
 
 # Use the resources from the android example
 set(RESOURCES
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/canary.wav
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/image.png
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/orchestral.ogg
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/sansation.ttf)
+    ${SRCROOT}/../android/assets/canary.wav
```
[the remark this answers is not in the document]
Comment (eXpl0it3r) [reply examples/iOS/CMakeLists.txt]: Consistency is the key, so I'd align it.

[the remark this answers is not in the document]
Comment (Ceylo) [reply examples/iOS/CMakeLists.txt]: I agree with sharing resources but this is maybe a waste of time depending on the output of https://en.sfml-dev.org/forums/index.php?topic=23886.0 (which may remove the iOS example as it is now). I'd recommend at least waiting for the output of the topic if you want to apply these consistency changes.

File: examples/iOS/CMakeLists.txt [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 # All source files
 set(SRC ${SRCROOT}/main.cpp)
 
 # Use the resources from the android example
 set(RESOURCES
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/canary.wav
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/image.png
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/orchestral.ogg
-    ${CMAKE_CURRENT_SOURCE_DIR}/../android/assets/sansation.ttf)
+    ${PROJECT_SOURCE_DIR}/examples/android/assets/canary.wav
```
Comment (binary1248) [inline examples/iOS/CMakeLists.txt]: I might have expressed myself a bit unclear in my last comment. I think it isn't too good of an idea to cross-reference resources from another example. To be completely clean, the iOS example should have a resources subdirectory with all its required resources like all the other examples. I would simply just copy over the files from the android assets directory into a new resources subdirectory so the final directory structure matches that of the other examples.

**Comment by eXpl0it3r** [comment]
I assume it's now okay, @binary1248?

**Comment by binary1248** [comment]
@eXpl0it3r Yes.

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Comment by eXpl0it3r** [comment]
Merged in 5ebfef4803a6b0f9b36972ce081f5674d62157ec

**Pull request closed without merging**